src/table/__tests__/empty-state.test.tsx

```diff
@@ -61,19 +62,49 @@ function findMergedCell(wrapper: TableWrapper) {
   return wrapper.findByClassName(styles['cell-merged']);
 }
 
+function mockResizeObserver(contentBoxWidth: number) {
+  const cbRef: { current: (entry: ContainerQueryEntry) => void } = { current: () => {} };
+  jest.mocked(useResizeObserver).mockImplementation((_target, cb) => {
+    // The table uses more than one resize observer.
+    // The callback must be triggered for all to ensure the expected one is targeted as well.
+    const prev = cbRef.current;
+    cbRef.current = entry => {
+      prev(entry);
+      cb(entry);
+      return 5;
+    };
```

Comment:
1. Same comment about using a list of callbacks instead of recursive chains: https://github.com/cloudscape-design/components/pull/1745#discussion_r1422909903 Think about your own debug experience and more readable stack traces
2. What does this number 5 do? In the types, you are saying it returns `void`

Reply:
Thanks. The number 5 return was for debugging, I forgot to get rid of it